app/V1Module/presenters/AssignmentSolutionsPresenter.php

```diff
@@ -274,11 +338,11 @@ public function checkSetAcceptedSubmission(string $id) {
   /**
    * Set solution of student as accepted, this solution will be then presented as the best one.
    * @POST
-   * @param string $id identifier of the submission
+   * @param string $id identifier of the solution
    * @throws \Nette\Application\AbortException
    * @throws NotFoundException
    */
-  public function actionSetAcceptedSubmission(string $id) {
+  public function actionSetAccepted(string $id) {
```

Comment:
Maybe also denote it as deprecated?

Reply:
You mean in doc comment? Or do we have some other ways?

Reply:
Doc comment. We have already done this ... search @DEPRECATED.